arm/datalake-analytics/job/client.go

```diff
@@ -0,0 +1,583 @@
+// Package job implements the Azure ARM Job service API version
```

Comment:
datalake-analytics Job is not an management service. It should not exists in the ARM namespace.